cmd/restore.go

```diff
@@ -40,8 +40,20 @@ func restoreCmd(passedExecs execs, cmdConfig *cmdConfiguration) (*cobra.Command,
 			}()
 			ctx = util.ContextWithTracer(ctx, tracer)
 			_, startupSpan := tracer.Start(ctx, "startup")
-			targetFile := args[0]
-			target := v.GetString("target")
+
+			// Get target from args[0], --target flag, or DB_RESTORE_TARGET environment variable
+			var target string
+			if len(args) > 0 {
+				target = args[0]
+			} else {
+				target = v.GetString("target")
+			}
+			if target == "" {
+				return fmt.Errorf("target must be specified as argument, --target flag, or DB_RESTORE_TARGET environment variable")
+			}
+
+			// Always pass empty targetFile to use the full path from the URL
+			targetFile := ""
```

Comment:
Instead of just setting `targetFile := ""` and passing empty strings around, should we completely remove the `TargetFile` field from the entire codebase?